### PR TITLE
nest options

### PR DESCRIPTION
**Pull request by purge**
sqwidget-css-moo-baa="gorch" becomes

`{ "css": { "moo": { "baa": "gorch } } }`

in config

### CODE REVIEW

File: app/main.js [hunk 1]
```diff
@@ -9,23 +9,33 @@ function(require, bonzo, qwery, bean) {
...
+          data = data[ names[i].toLowerCase() ] =
+            i === names.length - 1 ? val : data[ names[i] ] || {};
         }
-        key = key.replace("sqwidget", "").toLowerCase();
-        data[key || "url"] = val;
+      };
+
+      for (key in elData) {
+        val = elData[key];
+        if (!(key.match("^sqwidget"))) { continue; }
```
Comment (adhipg) [inline app/main.js:32]: I'm thinking if we should make all the data attributes available to the config? There's no harm in it and may keep embed codes shorter for someone.

I won't want to use something that's not prefixed for my own projects - but someone may.

Comment (purge) [inline app/main.js:32]: w3c recommends prefixing.

Comment (adhipg) [inline app/main.js:32]: So should we make this prefix a little more configurable? #97

File: app/main.js [hunk 2]
```diff
@@ -9,23 +9,33 @@ function(require, bonzo, qwery, bean) {
...
+      var key, val,
+          data = {},
+          elData = $el.data();
+
+      if(elData.sqwidget) {
+        elData.sqwidgetUrl = elData.sqwidget;
+        delete elData.sqwidget;
+      }
+
+      var nest = function( names, data, val ) {
```
Comment (adhipg) [inline app/main.js]: This definitely needs to be documented.

**Pull request merged**